doc/release-notes.rst

```diff
@@ -30,6 +30,12 @@ Documentation
 * Documentation refactorization with examples. :issue:`472` by :user:`Guillaume
   Lemaitre <glemaitre>`.
 
+Maintenance
+...........
+
+* Remove :class:`pydicom.filereader.DicomIter` since it is unused. :issue:`482`
+  by :user:`Guillaume Lemaitre <glemaitre>`.
+
```

Comment:
to be honest I just repeat what is done in scikit-learn. I would almost be fine to not mention credit since following the PR number you will know the different contributor. 

I am not against changing the way to write the entry, I am only interesting about consistency.